cmd/fyne/internal/commands/build.go

```diff
@@ -28,7 +28,7 @@ func Build() *cli.Command {
 	return &cli.Command{
 		Name:        "build",
 		Usage:       "Build an application.",
-		Description: "You may specify the --executable to build, but --target will define the OS it is build for indiscriminate of the extention of the executable.",
+		Description: "You may specify the -o to build, but --target will define the OS it is build for indiscriminate of the extention of the output file.",
```

Comment:
This sentence does not make sense. It looks like it was copied from package and manipulated to change the variable names. But it's content is still based on specifying an input executable.
How about something like the following?

`You can specify --target to define the OS to build for. The executable file will default to an appropriate name but can be overridden using -o`